### DISCUSSION TITLE
Vanilla Custom Item Integration

### DISCUSSION THREAD

**Discussion by ghost** [shedaniel/RoughlyEnoughItems#1289] [Q & A]
Hello! I wondered if I could customize REI to have a library of vanilla-created custom items in its entries.
I am playing on a vanilla server with custom items. I could add them to my favourite sidebar to keep track of their stats, but is there a way I could add items to the entry list en masse instead of the favourite bar if I have the NBT info?

**Comment by ItWasEnder** [comment]
I would also like to see this. Like a file that will just add custom items in to JEI instead of parsing from the crafting book.
